packages/react-devtools-shared/src/devtools/views/Profiler/ChartNode.js

```diff
@@ -55,7 +57,7 @@ export default function ChartNode({
         onDoubleClick={onDoubleClick}
         className={styles.Rect}
         style={{
-          opacity: isDimmed ? 0.5 : 1,
+          opacity: isHovered ? 0.75 : isDimmed ? 0.5 : 1,
```

Comment:
This doesn't work the way it's intended, I think. The way it's written, `isHovered` does not mean "this fiber is hovered" it just means "something is hovered" - so currently, everything dims at the same time, making the effect not useful.